.editorconfig

```diff
@@ -13,3 +13,5 @@ trim_trailing_whitespace = false
 [{.travis.yml,npm-shrinkwrap.json,package.json}]
 indent_style = space
 indent_size = 2
+[Makefile]
+indent_style = tab
```

Comment:
@SISheogorath @jackycute Tab character has special meaning for Makefile.
So it is proper to use tab indent instead of space in Makefile.
See [here](http://editorconfig.org/)